hal/src/thumbv7em/sercom/v2/spi.rs

```diff
@@ -538,10 +538,10 @@ pub type PadsFromIds<S, I, DI = NoneT, DO = NoneT, CK = NoneT, SS = NoneT> = Pad
 pub trait PadSet: Sealed {
     type Sercom: Sercom;
     type IoSet: IoSet;
-    type DataIn: OptionalPad;
-    type DataOut: OptionalPad;
-    type Sclk: OptionalPad;
-    type SS: OptionalPad;
+    type DataIn: OptionalPin;
+    type DataOut: OptionalPin;
+    type Sclk: OptionalPin;
+    type SS: OptionalPin;
```

Comment:
I think you didn't update something correctly. This should definitely be `OptionalPad`. Looks like there are problems throughout this module.

Reply:
Something weird happened with git. Should be fixed now.